src/sql/src/catalog.rs

```diff
@@ -1325,6 +1325,51 @@ impl Display for ObjectType {
     }
 }
 
+impl RustType<proto::ObjectType> for ObjectType {
```

Comment:
TIL we have two different definitions of "ObjectType", not that we need to do this here, but maybe we should unify them?